_posts/2018-08-15-Episode-0-Bokeh.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: post
+title: Open Source Directions Episode 0: Bokeh
+---```
+
+### Introductions
+
+This is the webinar that informs the community about what is going on with Open Source Python projects.  I am Anthony Scopatz and today the project being highlighted is Bokeh.
+
+Tony Fast joins us as a community organizer and open source consultant.
+
+Bryan Vandavin is our second guest, and he works for Anaconda and was a developer of conda, but has since begun working on Bokeh.
+
+### Headlines
+
+Just for a quick overview of the headlines; a security fix was applied to Jupyter notebook and Jupyter labs, so we would encourage everyone to update.  Scipy 2018 has ended, but many of the talks and additional info about Numpy can be found online.  The lightning talks were a great highlight, and especially noteworthy was when Guido van Rossum got up and they had a standing ovation for the end of his long vacation.
+
+### Project Overview
+
+At the moment Bokeh is a popular module, and Tony will summarize the stats from github.  It stands at upwards of 190,000 downloads per month, with 300 contributors, and around 7900 stars.  
+
+Now Anthony and Tony will ask Bryan some questions and gain some background; 
+
+Q. First I would like to ask, why was Bokeh started?  
+  A. It started 2012 with Peter Chang and the idea was that people wanted a rich visualization library that could work well with the web.  The intention was that it could be something like an updated version of chalco but more user friendly.  
```

Comment:
There should probably be a blank line between the question and answer.  And I actually think this should be less Q&A and more narative. "Bokeh was started in 2012..."

Reply:
I removed the question element from some things but kept it for the actual QA portion.  Not sure if that is alright.

Reply:
Yes, I think that is correct